### DISCUSSION TITLE
chapter_divide_and_conquer/

### DISCUSSION THREAD

**Discussion by giscus** [krahets/hello-algo#618] [Announcements]
# chapter_divide_and_conquer/

动画图解、一键运行的数据结构与算法教程

https://www.hello-algo.com/chapter_divide_and_conquer/

**Comment by BlackCat-12** [comment]
新的内容？佬加油

**Reply by krahets** [reply]
对呀，冲冲

**Reply by harisonkhlil** [reply]
冲

**Comment by Phoenix0415** [comment]
这张图太漂亮了

**Comment by wang-yulin** [comment]
大佬这个图配得真是恰到好处，能弱弱地问下是用的哪个AI作图工具吗？

**Comment by PeppaPige** [comment]
Cover nice

**Comment by MambaFly** [comment]
每章的封面图真好看

**Comment by MambaFly** [comment]
算法真暗含着人生哲学的意味，做事情做项目不也是分而治之吗，把大工程复杂任务逐步拆分到不可分解的子任务（一次就可完成的行动），每次专注于当前的子任务，最后汇总成终极目标

**Comment by masterzrz** [comment]
第一次用这本书，我也是深刻体会到程序员具备的有优良精神--开源精神

**Reply by yuechao4039** [reply]
还有分享，乃至奉献。感谢suski

**Comment by xiaozhuge123** [comment]
day10
